ovos_plugin_manager/thirdparty/solvers.py

```
@@ -107,7 +107,7 @@ def sentence_split(text: str, max_sentences: int = 25) -> List[str]:
                                  for t in text.split("\n")])[:max_sentences]
         except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)